### PR TITLE
Add memcached templates

### PR DESCRIPTION
**Pull request by arsenvlad**

### CODE REVIEW

File: memcached-multi-vm-ubuntu/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+            }
+        },
+        "virtualNetworkName": {
+            "type": "string",
+            "defaultValue": "vnet",
+            "metadata": {
+                "Description": "Virtual network name"
+            }
+        },
+        "addressPrefix": {
```
Comment (singhkay) [inline memcached-multi-vm-ubuntu/azuredeploy.json]: These and parameters below are missing description.

File: memcached-multi-vm-ubuntu/metadata.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Multiple Ubuntu 14.04 VMs with memcached service and one VM with Apache and PHP test page",
```
Comment (singhkay) [inline memcached-multi-vm-ubuntu/metadata.json]: Suggestion: A better title when this template gets displayed in azure.com might be something along the lines of "Memcached service cluster using multiple Ubuntu VMs" which puts the focus on the application that this template installs.

**Comment by singhkay** [comment]
@arsenvlad Left a few comments, thanks!

**Comment by arsenvlad** [comment]
@singhkay  Applied your suggestions, thanks!

**Comment by singhkay** [comment]
Thanks!

**Pull request merged**